code/modules/food_and_drinks/food/foods/misc_food.dm

```diff
@@ -66,16 +66,116 @@
 /obj/item/reagent_containers/food/snacks/aesirsalad
 	name = "aesir salad"
 	desc = "Probably too incredible for mortal men to fully enjoy."
+	icon = 'icons/obj/food/soupsalad.dmi'
 	icon_state = "aesirsalad"
 	trash = /obj/item/trash/snack_bowl
 	filling_color = "#468C00"
 	bitesize = 3
 	list_reagents = list("nutriment" = 8, "omnizine" = 8, "vitamin" = 6)
 	tastes = list("divinity" = 1, "lettuce" = 1)
 
+/obj/item/reagent_containers/food/snacks/antipasto_salad
+	name = "antipasto salad"
+	desc = "A traditional Italian salad made of salami, mozzarella cheese, olives, and tomatoes. Often served as a first course meal."
+	icon = 'icons/obj/food/soupsalad.dmi'
+	icon_state = "antipasto_salad"
+	trash = /obj/item/trash/snack_bowl
+	filling_color = "#468C00"
+	bitesize = 3
+	list_reagents = list("nutriment" = 12, "protein" = 6, "vitamin" = 6)
+	tastes = list("lettuce" = 2, "salami" = 2, "mozzarella cheese" = 2, "tomatoes" = 2, "dressing" = 1)
+
+/obj/item/reagent_containers/food/snacks/caesar_salad
+	name = "Caesar salad"
+	desc = "A simple yet flavorful salad of onions, lettuce, croutons, and shreds of cheese dressed in oil. Comes with a slice of pita bread!"
+	icon = 'icons/obj/food/soupsalad.dmi'
+	icon_state = "caesar_salad"
+	trash = /obj/item/trash/snack_bowl
+	filling_color = "#468C00"
+	bitesize = 3
+	list_reagents = list("nutriment" = 12, "vitamin" = 6)
+	tastes = list("healthy greens" = 2, "olive dressing" = 2, "feta cheese" = 2, "pita bread" = 1)
+
+/obj/item/reagent_containers/food/snacks/fruitsalad
```

Comment:
A lot of the salads you're adding have this stuff in common. No need to touch old foods, but could you make all of these salads into a new path? A lot of the cooking stuff like this sucks at inheritence
```suggestion
/obj/item/reagent_containers/food/snacks/salad/fruit
```